requirements.json

```diff
@@ -0,0 +1,194 @@
+{
+  "cdk": {
+    "name": "Red Hat Container Development Kit",
+    "description": "Developer Tools for Creating, Testing, and Distributing Red Hat Container-Based Applications",
+    "vendor": "Red Hat, Inc.",
+    "platform": {
+      "win32": {
+        "bundle": "yes",
+        "version": "3.0.0.GA",
+        "dmUrl": "http://cdk-builds.usersys.redhat.com/builds/nightly/11-Feb-2017/windows-amd64/minishift.exe",
+        "url": "http://cdk-builds.usersys.redhat.com/builds/nightly/11-Feb-2017/windows-amd64/minishift.exe",
+        "filename": "minishift.exe",
+        "sha256sum": "984318638a5c0d67ae4483bb508f6b90b99f885153dc525660bfff07fa883c94"
+      },
+      "darwin": {
+        "bundle": "yes",
+        "version": "3.0.0.GA",
+        "dmUrl": "http://cdk-builds.usersys.redhat.com/builds/nightly/11-Feb-2017/darwin-amd64/minishift",
+        "url": "http://cdk-builds.usersys.redhat.com/builds/nightly/11-Feb-2017/darwin-amd64/minishift",
+        "filename": "minishift",
+        "sha256sum": "1e7472175e25d9d197149a3ed2192440a1f581f1884cf2b2c77d97674ee73e5e"
+      },
+      "linux": {
+        "bundle": "yes",
+        "version": "3.0.0.GA",
+        "dmUrl": "http://cdk-builds.usersys.redhat.com/builds/nightly/11-Feb-2017/linux-amd64/minishift",
+        "url": "http://cdk-builds.usersys.redhat.com/builds/weekly/25-Jan-2017.3.0_alpha.2/linux-amd64/minishift",
+        "filename": "linux-amd64.tar.gz",
+        "sha256sum": "76da07d4065eb387093af1316371a19c11982b4b9270eff14bca84db69e2bb32"
+      }
+    }
+  },
+  "cygwin": {
+    "name": "Cygwin",
+    "description": "A distribution of popular GNU and other Open Source tools running on Microsoft Windows",
+    "vendor": "Community Project",
+    "platform": {
```

Comment:
I can't see any difference between the platforms here. Could it be done the way devstudio is?